src/IssueCreator/IssueManager.cs

```diff
@@ -37,18 +38,21 @@ public static IssueManager Create(Settings settings, string cacheFolder)
             return manager;
         }
 
-        private IssueManager(string _cache)
+        private IssueManager(string cache, FileLogger logger)
         {
-            _cacheFolder = _cache;
+            _cacheFolder = cache;
+            _fileLogger = logger;
         }
 
         public void RefreshZenHubToken(string newToken)
         {
+            _fileLogger.Log("Refreshing ZenHub token");
```

Comment:
Maybe not here, but below: it can be nice to know when it's done. Some solutions I've seen and used actually have a "scope". For example, you could return a disposable that writes "BEGIN: " and "END: " like so:

```csharp
public IDisposable LogStart(string message);
```

Set `<LangVersion>latest</LangVersion>` in your .csproj and you can use the simple `using IDisposable log = _fileLogger.Log("Refreshing ZenHub token");` without brackets.

Just a thought.